src/nprintml/learn/automl.py

```diff
@@ -85,6 +85,11 @@ def __call__(self, test_size=TEST_SIZE, eval_metric=EVAL_METRIC, quality=QUALITY
                                n_threads, verbosity=verbosity)
         self.test(predictor, test_data)
         self.graph_all(predictor, test_data)
+        self.save_features()
+
+    def save_features(self):
+        print(self.outpath)
+        self.data.to_csv(self.outpath / 'automl-df.csv.gz')
```

Comment:
This works perfectly fine, and makes a certain amount of sense, of course; but, I think it makes more sense for the label-aggregation step to take responsibility for the disk-caching of its own product, rather than throwing it on here.

It seemed easiest to demonstrate this via a separate pull request – #33 – and to add test assertions and the CLI flag, there.